components/calendar/tests/reference_year.rs

```diff
@@ -39,6 +43,9 @@ where
     for month_number in 1..=14 {
         for is_leap in [false, true] {
             for day_number in 1..=32 {
+                if !valid_md_condition(month_number, is_leap, day_number) {
```

Comment:
@sffc This test was unfortunately ignoring all dates that error, since it cannot tell the difference between "month day didn't occur in computed reference year" and "month day never occurs in calendar". I fixed it by introducing a condition.